src/WarcraftLegacies.Source/Quests/Legion/QuestAlteracBase.cs

```diff
@@ -16,19 +18,19 @@ public sealed class QuestAlteracBase : QuestData
     /// <summary>
     /// Initializes a new instance of the <see cref="QuestAlteracBase"/> class
     /// </summary>
-    public QuestAlteracBase(Rectangle rescueRect) : base("Ruins of Alterac",
-      "The orcs that occupied Alterac have maintained a secret demon gate, the Legion will make good use of it",
+    public QuestAlteracBase(Rectangle rescueRect, Capital caerDarrow) : base("Ruins of Alterac",
+      "The orcs that occupied Alterac have maintained a secret demon gate, the Legion will make good use of it. Thanks to their orc servants, the legion was able to propagate grim shrines around Lordaeron.",
       @"ReplaceableTextures\CommandButtons\BTNDemonCrypt.blp")
     {
-      AddObjective(new ObjectiveControlPoint(
-        ControlPointManager.Instance.GetFromUnitType(Constants.UNIT_N019_ALTERAC_MOUNTAINS)));
+      AddObjective(new ObjectiveControlCapital(caerDarrow, false));
 
       _rescueUnits = rescueRect.PrepareUnitsForRescue(RescuePreparationMode.HideAll);
       Required = true;
+      ResearchId = Constants.UPGRADE_R01Q_QUEST_COMPLETED_RUINS_OF_ALTERAC;
     }
 
     /// <inheritdoc/>
-    protected override string RewardDescription => "Control and reveal a small base in Alterac";
+    protected override string RewardDescription => "Gain control of a small base in Alterac, and learn to generate a portal to Alterac using the Argus Teleporter. Also gain 1 Dread Shrine in three Scourge bases around Lordaeron.";
```

Comment:
```suggestion
    protected override string RewardDescription => "Gain control of a small base in Alterac, learn to generate a portal to Alterac using the Argus Teleporter, and gain a Dread Shrine in each of the following Scourge bases: x, y, and z";
```
Should also be using `GetObjectName` for the Dread Shrine. Technically the Argus Teleporter too but that's whatever.